utils/test.js

```diff
@@ -16,3 +16,89 @@ exports.mockCalendlyAuthentication = () => {
       },
     });
 };
+
+exports.mockCalendlyScheduledEvents = () => {
+  nock(CALENDLY_API_BASE_URL)
+    .persist()
+    .get('/scheduled_events')
+    .reply(200, {
+      resource: {
+        collection: [],
+        pagination: {},
+      },
+    });
+};
+
+exports.mockCalendlyEventTypes = () => {
+  nock(CALENDLY_API_BASE_URL)
+    .persist()
+    .get('/event_types')
+    .reply(200, {
+      resource: {
+        collection: [],
+        pagination: {},
+      },
+    });
+};
+
+exports.mockCalendlyEvent = () => {
+  const uuid = 'someUuid';
+
+  nock(CALENDLY_API_BASE_URL)
+    .persist()
+    .get(`/scheduled_events/${uuid}`)
+    .reply(200, {
+      resource: {
+        uri: 'Event uri',
+      },
+    });
+};
```

Comment:
This would be a really nice mock to put a parameter on.
```suggestion
exports.mockCalendlyEvent = (uuid = 'someUuid') => {
  nock(CALENDLY_API_BASE_URL)
    .persist()
    .get(`/scheduled_events/${uuid}`)
    .reply(200, {
      resource: {
        uri: 'Event uri',
      },
    });
};
```

Reply:
Indeed! Ha, that's better. Thank you!